app/settings/addons/controller.ts

```diff
@@ -1,8 +1,14 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
+import Media from 'ember-responsive';
 
 import CurrentUser from 'ember-osf-web/services/current-user';
 
 export default class SettingsAddonsController extends Controller {
     @service currentUser!: CurrentUser;
+    @service media!: Media;
+
+    get isMobile(): boolean {
+        return this.media.isMobile;
+    }
```

Comment:
Fun fact: ember-responsive has a `media` helper that you can use like `{{#if (media 'isDesktop')}}`. We should only need to custom define something if we need it to cover multiple breakpoints at once (like mobile and tablet), though we could also use an `or` there I suppose as well.